skgstat/Variogram.py

```diff
@@ -2383,6 +2384,39 @@ def nrmse(self):
         """
         return self.rmse / np.nanmean(self.experimental)
 
+    @property
+    def root_mean_square(self):
+        """Root Mean Square (RMS) of the residuals
+
+        Calculates the square root of the mean of squared residuals.
+
+        Returns
+        -------
+        float
+            Root Mean Square of the residuals.
+        """
+        return np.sqrt(np.nanmean(np.square(self.model_residuals)))
```

Comment:
Sorry if I'm blind right now, but is this not the same as `Variogram.rmse`?
We could update the the existing rmse with this implementation, which is way cleaner; or let `Variogram.rmse` just call your function.

Reply:
It seems like mean square of the residuals (RMS) could be the Root Mean Square Error (RMSE), however in the book [1](https://link.springer.com/book/10.1007/978-3-319-15865-5) its not explicitaly menctioned ~ which is why I added it separately; similarly here is used with the same nomenclature [2](https://www.researchgate.net/figure/Estimation-of-the-root-mean-square-RMS-of-the-residual-between-the-data-and-the-model_fig5_326735330). 

It could be a typo in the book (?) and its just the RMSE, for me leaving RMSE is fine..

Reply:
I pointed my old `Variogram.rmse` to your `Variogram.root_mean_square` and all the tests still work, so in fact, it is the same thing.
Then, we keep both with `Variogram.rmse` simply calling `Variogram.root_mean_square`, as it has the much cleaner implementation.